hudi-common/src/main/java/org/apache/hudi/common/model/OverwriteWithLatestAvroPayload.java

```diff
@@ -48,16 +48,7 @@ public OverwriteWithLatestAvroPayload(Option<GenericRecord> record) {
 
   @Override
   public OverwriteWithLatestAvroPayload preCombine(OverwriteWithLatestAvroPayload oldValue) {
-    if (oldValue.recordBytes.length == 0) {
-      // use natural order for delete record
```

Comment:
Instead of modifying the existing payload, can we write a new payload named `CommitTimeAvroPayload` akka to the renamed merger mode: `COMMIT_TIME_ORDERING`

Reply:
As we have discussed offline, will fix the related CI failures.